### ISSUE TITLE
Issues with second screen recognition

### ISSUE THREAD

**Issue by Wollipolli** [pdfpc/pdfpc#217]
Hi folks,

I really appreciate the time and effort you have put into this amazing project - I really love it. Finally I don't have to look up to powerpoint-users, but can use latex-beamer presentations in even more powerful ways than they do!

I had two issues before I could make it work on my machine, and was wondering, whether these are my responsibility, or could generally be fixed from pdfpc side.

I use Linux Mint 18 which is built on Ubuntu 16.04, and the canonical version of pdfpc v4.0.2 which is certainly deprecated, as almost all packages for Ubuntu ...  I have a Dell laptop sitting on a dock, which has 2 screens permanently attached and the laptop's display as 3rd screen and primary. [laptop][2][3]

1. My monitor setup was not recognized. Every time I start pdfpc - both windows (the presenter and the presentation) are shown on my primary screen. I have to manually drag the Presentation to the other display, which is possible, but annoying! For the sake of testing, if this is an issue due to three screens attached, I also disconnected one of the screens, but this did not change anything. I remember, that the setup of the screens was not straight-forward - maybe I changed some config, so that pdfpc can now not recognize the other screens. Attached my xrandr config:
```
Screen 0: minimum 8 x 8, current 5120 x 1715, maximum 32767 x 32767
eDP1 connected primary 1920x1080+0+635 (normal left inverted right x axis y axis) 309mm x 173mm
   1920x1080     60.00*+  59.93    48.00  
   1680x1050     59.95    59.88  
   1600x1024     60.17  
   1400x1050     59.98  
   1600x900      60.00  
   1280x1024     60.02  
   1440x900      59.89  
   1280x960      60.00  
   1368x768      60.00  
   1360x768      59.80    59.96  
   1152x864      60.00  
   1280x720      60.00  
   1024x768      60.00  
   1024x576      60.00  
   960x540       60.00  
   800x600       60.32    56.25  
   864x486       60.00  
   640x480       59.94  
   720x405       60.00  
   640x360       60.00  
DP1 disconnected (normal left inverted right x axis y axis)
DP1-1 connected 1920x1200+1920+0 (normal left inverted right x axis y axis) 518mm x 324mm
   1920x1200     59.95*+
   1920x1080     60.00  
   1600x1200     60.00  
   1680x1050     59.88  
   1280x1024     60.02  
   1280x960      60.00  
   1024x768      60.00  
   800x600       60.32  
   640x480       60.00  
   720x400       70.08  
DP1-2 connected 1280x1024+3840+141 (normal left inverted right x axis y axis) 376mm x 301mm
   1280x1024     60.02*+  75.02  
   1152x864      75.00  
   1024x768      75.08    60.00  
   800x600       75.00    60.32  
   640x480       75.00    60.00  
   720x400       70.08  
DP1-3 disconnected (normal left inverted right x axis y axis)
HDMI1 disconnected (normal left inverted right x axis y axis)
HDMI2 disconnected (normal left inverted right x axis y axis)
VIRTUAL1 disconnected (normal left inverted right x axis y axis)

```
2. I find it strange to always have to invoke pdfpc with the -n<pos> option. But otherwise my latexc-note slides are not separated from the presentation. Is that the only way to do it?

Cheers, and keep up the good work!
Wolle

**Comment by AndreasBilke** [comment]
So do I understand it correctly, your bug report represents problems with v4.0.2 ?
1. It should also work with three monitors. The presenter view should be on the primary and the presentation on one other screen. What WM do you use? We did a lot of work to make moving windows better in v4.0.6 (especially with different WMs)
2. Also with v4.0.6 you only have to add -n once for each file. Then the option is added to the `pdfpc` file and is used for further pdfpc invocations.

**Comment by Wollipolli** [comment]
Yes, I use 4.0.2 - but I can build and try a more recent version tomorrow at work ;).

1. I use Cinnamon.
2. Ah I did not know that - but maybe a global option for all instances of pdfpc for me will make more sense, since I will only use it in that setup, but for several presentations, of course.

**Comment by phillipberndt** [comment]
What you describe is exactly my configuration, and I suffer from the same
bug. But this is a cinnamon issue, not one on pdfpc's end. So don't bother
testing or trying to fix this in pdfpc. Even manually issued move requests
to the presentation window aren't processed correctly (for example, a move
to 0x0 moves the window to the second screen here). If I only use an
external beamer and the notebook screen things work as expected, so this
isn't really a big issue.

Wollipolli <notifications@github.com> schrieb am Di., 28. Feb. 2017, 16:20:

**Comment by Wollipolli** [comment]
I looked a litte deeper into the cinnamon - screen detection issue:
1. Hooking the laptop to a dock with a projector connected, does work out of the box, recognizes primary and presentation screen and displays the presentation in the correct dimensions.
2. Back at my desk, I tried to find the location cinnamon saves the display setup. According to [this website](http://segfault.linuxmint.com/2015/08/configuring-the-mdm-login-screen-for-multiple-monitors/) it's in `~/.config/monitors.xml`. The file looked pretty messy, having several configurations for my screens. I reduced it to the version I am basically seeing, when typing xrandr. But this didn't change anything. 
3. Then I tried to add a correct xrandr command with the setup to the mdm config file (`/etc/mdm/Init/Default`) but this also did not change anything.
